components/KidsList/KidsList.js

```diff
@@ -0,0 +1,32 @@
+import useSWR from "swr";
+import { StyledHeading, StyledList, StyledListContainer,
+    StyledListItem, StyledImage, StyledParagraph } from "./KidsList.styled";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+export default function KidsList() {
+  const { data, isLoading } = useSWR("/api/kids", fetcher);
```

Comment:
You already have the `fetcher` in the SWRConfig ^^

Reply:
Right.